dali/benchmark/CMakeLists.txt

```diff
@@ -36,12 +36,12 @@ if (BUILD_BENCHMARK)
     list(APPEND DALI_BENCHMARK_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/caffe2_alexnet_bench.cc")
   endif()
 
-  set(benchmark_bin "dali_benchmark.bin")
-  add_executable(${benchmark_bin} "${DALI_BENCHMARK_SRCS}")
+  add_executable("dali_benchmark.bin" "${DALI_BENCHMARK_SRCS}")
```

Comment:
Nitpick:
I think the most elegant way of specifying this name would be to use unquoted target name (not necessarily the file name) and specify the file name in set_target_properties - but I'm not pushing it; maybe it would break something with the wheel building etc.

Reply:
Done

Reply:
Done